packages/qdrant-loader/src/qdrant_loader/cli/commands/jobs_cmd.py

```
@@ -0,0 +1,239 @@
+
+import asyncio
+import json
+from pathlib import Path
+
+import click
+from click.exceptions import ClickException
+from click.types import Choice
+from click.types import Path as ClickPath
+
+
+def _init_queue(workspace: Path | None, config: Path | None, env: Path | None):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_init_queue`

(ANN202)